make/wag.mk

```diff
@@ -23,7 +23,7 @@ bin/wag: ensure-wag-version-set
 
 jsdoc2md:
 	hash npm 2>/dev/null || (echo "Could not run npm, please install node" && false)
-	hash jsdoc2md 2>/dev/null || npm install -g jsdoc-to-markdown@^2.0.0
+	test -f ./node_modules/.bin/jsdoc2md || npm install jsdoc-to-markdown@^2.0.0
```

Comment:
Do you want to change the line above to `test -f`?

Reply:
to use `test` we'd need to know the exact location of `npm` in the user's path, which I don't think we know for sure.